### DISCUSSION TITLE
not able to upgrade from 11.3.1

### DISCUSSION THREAD

**Discussion by dersupergrobi** [jomjol/AI-on-the-edge-device#2576] [Q&A]
Hey,

just wanted to upgrade from 11.3.1 an read back in the history.
so i tried to upgrade to 12.0.1 but it doenst work

Uploaded firmware bin to Firmware
Uplaoded the HTMP uzip file to html and rebootet

but still on 11.3.1

what is wrong in my process?

**Comment by dersupergrobi** [comment]
ok...was the worng process...did the last updates long ago

**Reply by caco3** [reply]
we do not test migration between such big version changes. But you can always directly flash and reuse the configuration.
